src/views/htmlcontent/src/css/color-theme-dark.css

```diff
@@ -103,5 +103,5 @@ slick-grid.active .grid .slick-cell.selected {
 }
 
 .shortCut {
-  color: grey;
+  color: rgb(255, 255, 255);
```

Comment:
I think you should use var(--color-content) instead. can you give it a try?

Reply:
sure.